### PR TITLE
Fix analyzer warnings in updater package

### PR DESCRIPTION
**Pull request by abarth**
This class was renamed to MojoEventSubscription.

### CODE REVIEW

**Comment by Hixie** [comment]
We should update `flutter analyze` to catch this, as follows:

In analyze.dart, copy the code that handles .../examples/*/lib/main.dart to also handle .../packages/*/lib/main.dart, in the block that current does .../packages/*/bin/*.dart.

**Comment by abarth** [comment]
Done.  PTAL

File: packages/flutter_tools/lib/src/commands/analyze.dart [hunk 1]
```diff
@@ -63,6 +63,13 @@ class AnalyzeCommand extends FlutterCommand {
         }
       }
 
+      String updaterPackagePath = path.join(ArtifactStore.flutterRoot, 'packages', 'updater');
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/analyze.dart]: I meant we should check all flutter/packages/*/lib/*/main.dart files, the same way we check all flutter/packages/*/bin/*.dart files.

**Comment by abarth** [comment]
Done.  PTAL

**Comment by Hixie** [comment]
LGTM

**Pull request merged**